src/main/java/be/seeseemelk/mockbukkit/block/state/BrewingStandMock.java

```diff
@@ -14,6 +15,7 @@ public class BrewingStandMock extends ContainerMock implements BrewingStand
 
 	private int brewingTime;
 	private int fuelLevel;
+	private BrewerInventoryMock inventory = new BrewerInventoryMock(this);
```

Comment:
This doesn't need its own field. `#createInventory` needs to be implemented and the ContainerMock will take care of it.